lib/vsc/install/shared_setup.py

```diff
@@ -501,6 +515,32 @@ def make_release_tree(self, base_dir, files):
             self.copy_file(os.path.join(REPO_BASE_DIR, fn), os.path.join(base_dir, fn))
 
 
+class vsc_sdist_rpm(vsc_sdist):
```

Comment:
all this code is modeled after disturils/setuptools, including non-standard naming convention